emily/handler/Cargo.toml

```diff
@@ -32,6 +34,10 @@ utoipa = "4.2.3"
 warp = "0.3"
 warp_lambda.workspace = true
 
+fake = { version = "2.9.2", features = ["derive", "time"], optional = true }
+
 [dev-dependencies]
 mockito = "0.28"
 mockall = "0.10"
+serial_test = "0.5"
+assert_unordered = "0.2"
```

Comment:
Why not use a library?

Reply:
I'm defaulting to libraries more because they seem to be so common in Rust, I don't want to go around reinventing these things. There's a huge benefit to readability if we're using common libraries.

Reply:
Yeah, libraries are often used in Rust-land. In particular, there are a few very commonly used libraries. For not-so-commonly used ones, I tend to not use them since I require myself to vet them. There have been a few trojan crates discovered on `crates.io` (they had a blog post about it a few months ago). These dependencies here seem okay (I did a basic vet of them and their dependencies and they looked largely fine, I wasn't super robust about it though). But yeah, I wouldn't go wild with the dependencies.


> I'm defaulting to libraries more because they seem to be so common in Rust, I don't want to go around reinventing these things. There's a huge benefit to readability if we're using common libraries.

Yeah, but I didn't think `assert_unordered` was a common library. I had to look it up (I did a basic vet of it too). Using `.sort_by` is also readable and easy to understand, and is in the standard library.


---

I'm not trying to harp on library use or even this one in particular. I like them, use them, blah blah blah. But I do want to increase the cost of adding them, and this seems like a case where I'd go without adding them (because I wouldn't want to vet it and the alternative is simple enough).

Reply:
Works for me

Reply:
👍🏽 cool. One last thing, I'd change this to
```toml
assert_unordered = { version = "0.2", default-features = false }
```
It doesn't pull in any transient dependencies if you don't use the default features.